keyboards/handwired/onekey/rp2040/mcuconf.h

```diff
@@ -10,3 +10,9 @@
 
 #undef RP_PWM_USE_PWM4
 #define RP_PWM_USE_PWM4 TRUE
+
+#undef RP_ADC_USE_ADC1
+#define RP_ADC_USE_ADC1 TRUE
+
+#undef RP_IRQ_ADC1_PRIORITY
+#define RP_IRQ_ADC1_PRIORITY 2
```

Comment:
Shall we move this into the default?

Reply:
Probably yes (we have 3 such files though).